tests/config_test.py

```
@@ -204,3 +205,231 @@ def load(f):
+            elif "tomllib" in sys.modules:
+                del sys.modules["tomllib"]
+
+            if original_tomli is not None:
+                sys.modules["tomli"] = original_tomli
+            elif "tomli" in sys.modules:
+                del sys.modules["tomli"]
+
+            # Force reimport of config module to restore original state
+            if "commit_check.config" in sys.modules:
+                del sys.modules["commit_check.config"]
+            import commit_check.config  # noqa: F401
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)